client/packages/plugins/README.md

```diff
@@ -161,8 +161,12 @@ The simplest way to begin is by cloning (forking for now or just copy and create
 ```
 git submodule add [your-plugin-bundle-repo-url] client/packages/plugins/myPluginBundle
 ```
+>   You will need to have github authentication set up to add restricted access repos from command line. [github cli](https://cli.github.com/) can conveniently set up github command line authentication access.  Other [alternative methods](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github) are also available.
 
-Note the `myPluginBundle` can be anything. The inner repository and core will be treated as two different repositories, changes in them will only be reflected in relative repositories (i.e. you can add the inner repository as local repository in github desktop). Make sure that you don't commit the `.gitmodule` or the single `client/packages/plugins/{your plugin bundle name}` to the core.
+With GitHub, I usually insert a blockquote.
```

Comment:
I don't understand this comment?

Reply:
Whoops yes this was me copy pasting markdown formatting comment from stack exchange to make my note!